frontend/src/pages/Register.jsx

```diff
@@ -2,17 +2,16 @@ import { useState } from "react";
 import { useNavigate, Link } from "react-router-dom";
 import { Loading } from "../reusables/Loading";
 import styled from "styled-components";
-import { EgoButton } from "../reusables/Button"
+import { EgoButton } from "../reusables/Button";
 import Globe from "../assets/Globe-ego.svg";
-import logo from "../assets/globe-logo.svg";
```

Comment:
same here - why not use the SVG here as well?

Reply:
Same answer as above